libs/geo/src/main/java/org/elasticsearch/geo/geometry/Circle.java

```diff
@@ -26,18 +26,25 @@
     public static final Circle EMPTY = new Circle();
     private final double lat;
     private final double lon;
+    private final double alt;
     private final double radiusMeters;
 
     private Circle() {
         lat = 0;
         lon = 0;
+        alt = Double.NaN;
         radiusMeters = -1;
     }
 
     public Circle(final double lat, final double lon, final double radiusMeters) {
+        this(lat, lon, radiusMeters, Double.NaN);
+    }
+
+    public Circle(final double lat, final double lon, final double radiusMeters, final double alt) {
```

Comment:
It seems more natural for me to have alt after [lat, lon] and radius the last parameter.